README.md

```diff
@@ -102,10 +102,56 @@ If the `--info` flag is provided to gradle it will output a dependency tree whic
 
 ![Tree](docs/Tree.PNG)
 
-Report output
+## Reporting in a Jenkins Pipeline
 -------------
 
-TODO
+The gradle plugin supports writing out test reports in the correct XML format for the
+[Jenkins JUnit Reporting Plugin](https://wiki.jenkins.io/display/JENKINS/JUnit+Plugin).
+To switch on this reporting, set the path to the report in you project's build.gradle file using the **"junitReport"** element like so:
+
+```
+audit {
+        failOnError = false
+        ignore = [ 'ch.qos.logback:logback-core' ]        ]
+        junitReport = "./ossindex/junitReport.xml"
+    }
+```
+
+This would create the file in an /ossindex folder in the project root.
+
+To access this using the JUnit plugin in a Jenkins pipeline:
+
+```
+    stages {
+
+        stage('OSSIndex Scan') {
+            steps {
+                sh "./gradlew --no-daemon audit"
+            }
+            post {
+                always {
+                    junit '**/ossindex/junitReport.xml'
+                }
+            }
+        }
+
+    }
+```
+
+NOTE: The junit plugin uses a slightly different syntax to reference the path.
+The touch command is there to get around the junit plugin complaining that the file

```

Comment:
I don't see the touch command, I assume it was removed?

Reply:
Yes, presumably a rush of crap to the brain on my part!

I've put it back.